package.json

```diff
@@ -21,5 +21,10 @@
   "devDependencies": {
     "redux-devtools": "^3.4.1"
   },
-  "proxy": "http://localhost:8080/"
+  "proxy": {
+    "/api": {
+      "target": "http://18.222.254.118:8080",
+      "changeOrigin" : true
+    }
+  }
```

Comment:
this is to connect to aws deployed back end, the changeOrigin true is for firefox, in my case the proxy didn't work without it.